modules/ossm-cert-manager-integration-istio.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+:_content-type: CONCEPT
+[id="ossm-cert-manager-integration-istio_{context}"]
+= About integrating Service Mesh with cert-manager and istio-csr
+
+cert-manager is a solution for X.509 certificate management on Kubernetes. It delivers a unified API to integrate applications with private or public PKIs, like Vault, Google Cloud Certificate Authority Service, Let's Encrypt, and more. 
+
+It ensures the certificates are valid and up-to-date, and attempts to renew certificates at a configured time before expiration. 
+
+cert-manager also provides integration for Istio users. `istio-csr` is a certificate authority (CA) server that handles certificate signing requests (CSR) from Istio proxies and delegates signing to cert-manager that forwards CSRs to the configured CA server.
+
+[NOTE]
+====
+Red Hat provides support for integrating with `istio-csr` and cert-manager. It does not provide direct support for the `istio-csr` or the community cert-manager components. The use of community cert-manager shown here is for demonstration purposes only.
+====
+
+.Prerequisites
+* One of these versions of cert-manager: 
+** {cert-manager-operator} 1.10 or later
+** community cert-manager Operator 1.11 or later
+** cert-manager 1.11 or later
+
+* OpenShift Service Mesh Operator 2.4 or later
+* `istio-csr` 0.6.0 or later
+
+[NOTE]
+====
+To avoid creating configmaps in all namesspaces when `istio-csr` is installed with helm chart `jetstack/cert-manager-istio-csr`, Red Hat recommends setting `app.controller.configmapNamespaceSelector: "maistra.io/member-of: <istio-namespace>`".
```

Comment:
```suggestion
To avoid creating config maps in all namespaces when the `istio-csr` server is installed with the `jetstack/cert-manager-istio-csr` Helm chart, use the following setting: `app.controller.configmapNamespaceSelector: "maistra.io/member-of: <istio-namespace>`".
```
- edit for clarity and styles. For example, "config map" or "the `ConfigMap` object".
- Also it's not clear to me where this setting needs to be changed. What file or config map?